sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -413,6 +413,78 @@ class DataFrameFunctionsSuite extends QueryTest with SharedSQLContext {
     )
   }
 
+  test("element at function") {
```

Comment:
why do we need so many test cases here? this is just to verify the api works end to end.

Reply:
also the function is element_at, not "element at" ...

Reply:
good catch, thanks